pkg/registry/apis/dashboard/legacysearcher/search_client.go

```diff
@@ -120,7 +123,68 @@ func (c *DashboardSearchClient) Search(ctx context.Context, req *resource.Resour
 			}
 
 			query.FolderUIDs = folders
+		case resource.SEARCH_FIELD_REPOSITORY_PATH:
+			// only one value is supported in legacy search
+			if len(vals) == 1 {
```

Comment:
what happens if vals length is greater than 1? should we emit an error or something?